### PR TITLE
Adds CLI commands and loading functions to assist with key management

### PR DESCRIPTION
**Pull request by thomastaylor312**
This was going to be part of the PR for adding verification to the client, but it had already gotten big enough that I decided this should be its own PR. See the individual commit messages for more specifics on what has been added

### CODE REVIEW

File: tests/cli.rs [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 use test_util::*;
 
 use bindle::client::{tokens::TokenManager, Client};
-use bindle::testing;
+use bindle::{testing, SignatureRole};
+
+const ENV_BINDLE_KEYRING: &str = "BINDLE_KEYRING";
+const KEYRING_FILE: &str = "keyring.toml";
+const SECRETS_FILE: &str = "secret_keys.toml";
+const TEST_LABEL: &str = "Benjamin Sisko <thesisko@bajor.com>";
```
Comment (technosophos) [inline tests/cli.rs:15]: 😆

**Review by technosophos — APPROVED** [review]
These are some much-needed fixes/refactors/improvements. 👍

File: tests/cli.rs (hunk 1 above)
Comment (radu-matei) [reply tests/cli.rs:15]: ❤️

**Comment by thomastaylor312** [comment]
Windows is still flaking on reading files from disk sometimes. I am not entirely sure why. I ran the tests on a local machine and manually checked that the failing test is a flake

**Pull request merged**